CRM/Activity/BAO/Activity.php

```diff
@@ -744,21 +743,55 @@ public static function getActivities($params) {
       'case_id' => 'case_id',
     ];
 
-    foreach ($result['values'] as $id => $activity) {
-
+    if (empty($result)) {
+      $targetCount = [];
+    }
+    else {
+      $targetCount = CRM_Core_DAO::executeQuery('
+      SELECT activity_id, count(*) as target_contact_count
+      FROM civicrm_activity_contact
+      INNER JOIN civicrm_contact c ON contact_id = c.id AND c.is_deleted = 0
+      WHERE activity_id IN (' . implode(',', array_keys($result)) . ')
```

Comment:
just thinking should this really be a param? not sure that you can actually inject anything but meh

Reply:
I tried but commaSeparatedInteger won't pass through in that way
- however it is definitely safe as they are the array keys from the api array